pyvrp/stop/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .MaxIterations import MaxIterations as MaxIterations
 from .MaxRuntime import MaxRuntime as MaxRuntime
+from .MultipleCriteria import MultipleCriteria as MultipleCriteria
```

Comment:
I'm open to suggestions for a better name!

Reply:
I like this one! I have tried before to come up with a name (for ALNS) but I think this one is clear enough. 

(Off-topic: In ALNS it should probably be called MultipleStoppingCriteria to avoid confusion with acceptance criteria.)